tests/baselines/reference/user/chrome-devtools-frontend.log

```diff
@@ -12750,7 +12750,12 @@ node_modules/chrome-devtools-frontend/front_end/ui/ContextMenu.js(339,39): error
 node_modules/chrome-devtools-frontend/front_end/ui/ContextMenu.js(340,39): error TS2339: Property 'y' does not exist on type 'Event'.
 node_modules/chrome-devtools-frontend/front_end/ui/ContextMenu.js(344,24): error TS2339: Property 'deepElementFromPoint' does not exist on type 'Event'.
 node_modules/chrome-devtools-frontend/front_end/ui/ContextMenu.js(352,24): error TS2694: Namespace 'Common' has no exported member 'Event'.
+node_modules/chrome-devtools-frontend/front_end/ui/ContextMenu.js(355,22): error TS2339: Property '_useSoftMenu' does not exist on type 'typeof ContextMenu'.
```

Comment:
Associated code is
```ts
  static initialize() {
    InspectorFrontendHost.events.addEventListener(InspectorFrontendHostAPI.Events.SetUseSoftMenu, setUseSoftMenu);
    /**
     * @param {!Common.Event} event
     */
    function setUseSoftMenu(event) {
      UI.ContextMenu._useSoftMenu = /** @type {boolean} */ (event.data); // this line
    }
  }
```

Looks like we're no longer recognizing namespace assignments within functions?

Reply:
Yep, it's #29335. @weswigham I'll create a bug and assign it to you although I don't have time to create a standalone repro.